freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_install.sh

```diff
@@ -3,7 +3,8 @@
 set -e
 
 FQDN=$(hostname -f)
-IPADDR=$(hostname -i)
+IPADDRS=$(hostname -i)
+IPADDR="${IPADDRS%% *}"
```

Comment:
could you add some comments how this expression works? also some log would be nice to make it easier to investigate if something goes sideways